packages/frontend/src/page/components/wallet-position/index.tsx

```diff
@@ -160,98 +165,109 @@ export const WalletPosition = ({
                         </Typography>
                     )}
                 </div>
-                <div className="w-full flex flex-col md:flex-row justify-between">
-                    <div className="w-full md:w-1/3 p-6 flex-col border-b border-black md:border-r">
-                        <Typography
-                            variant="xs"
-                            uppercase
-                            className={{ root: "mb-2" }}
-                        >
-                            {t("position.rewards.guaranteed.label")}
-                        </Typography>
-                        <div
-                            data-testid="wallet-position-guaranteed-rewards"
-                            className="flex flex-col gap-2"
-                        >
-                            {loading || !guaranteedRewards
-                                ? new Array(rewards?.length || 1)
-                                      .fill(null)
-                                      .map((_, index) => (
-                                          <TokenAmount key={index} loading />
-                                      ))
-                                : guaranteedRewards.map((reward) => {
-                                      return (
-                                          <TokenAmount
-                                              data-testid={`wallet-position-guaranteed-reward-${reward.currency.address}`}
-                                              key={reward.currency.address}
-                                              amount={reward}
-                                          />
-                                      );
-                                  })}
+                {!kpiToken.finalized && (
```

Comment:
Should we also hide this if the KPI is expired?

Reply:
Makes sense for sure I think. Addressed in ce0bfd3f9a80c4a726c573429522d1c5b06b85b7